tests/screenshot/apps/components/Header.js

```diff
@@ -0,0 +1,10 @@
+import Header from '../../../../Header';
+import React from 'react';
+
+const HeaderTests = [
```

Comment:
Shouldn't there be a title?

Reply:
Yes, there should. I fixed it, Thank you.